lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -202,32 +292,26 @@ div.course-wrapper {
         border-left: 4px solid #C93B34 !important;
         margin: 0 auto;
       }
+      &.warning {
+        border-left: 4px solid #FAB95C !important;
```

Comment:
Use color variable to keep this themeable: 
```
border-left: 4px solid $warning-color !important;
```
It would be good to drop all the !importants above but I see you are just adding a new case so I won't ask for that now.